util/private.num

```diff
@@ -340,7 +361,6 @@ OSSL_PARAM_TYPE                         generic
 OSSL_PARAM_construct_TYPE               generic
 OSSL_PARAM_octet_string                 define
 OSSL_PARAM_utf8_string                  define
-OSSL_PARAM_octet_ptr                    define
```

Comment:
how did this happen?

Reply:
This is a secondary correction of glitch (double entry) in the master.
Well, so this should more cleanly be done in a separate PR, 
but on the other hand that hardly worth the extra effort.

Reply:
okay if this can be merged soon. I have currently viewed 30 out of 38 files...

Reply:
We could also simply neglect this issue and remove the fix done here.

Reply:
If it's in a separate commit, without fixup! comment, then it's okay for me.

Reply:
All right, done.